Content.Shared/Doors/Systems/SharedFirelockSystem.cs

```diff
@@ -48,18 +41,7 @@ public bool EmergencyPressureStop(EntityUid uid, FirelockComponent? firelock = n
         return _doorSystem.OnPartialClose(uid, door);
     }
 
-    #region Access/Prying
-
-    private void OnBeforeDoorOpened(EntityUid uid, FirelockComponent component, BeforeDoorOpenedEvent args)
-    {
-        // Give the Door remote the ability to force a firelock open even if it is holding back dangerous gas
-        var overrideAccess = (args.User != null) && _accessReaderSystem.IsAllowed(args.User.Value, uid);
-
-        if (!component.Powered || (!overrideAccess && component.IsLocked))
-            args.Cancel();
-        else if (args.User != null)
-            WarnPlayer((uid, component), args.User.Value);
-    }
+    #region Prying
```

Comment:
It will warn an engineer that the firelock is locked, without warning about the specific danger (see video).

It is an option to bring back the pressure-specific warning, but see my discussion about the flaws of the pressure-differential detection [here](https://forum.spacestation14.com/t/air-alarm-and-firelock-design/9325).